spec/tapioca/cli/gem_spec.rb

```diff
@@ -1022,11 +1022,9 @@ module Foo
           @project.require_mock_gem(foo)
           # We know RDoc is problematic with respect to calling `abort` in an autoload
           @project.require_real_gem("rdoc")
-          # Just so that we have something else we can generate for.
-          @project.require_real_gem("json")
```

Comment:
I removed `json` as an example here, as it added a new `gem "json"` line to the Gemfile, which conflicts with the `gem "json", ">= 2.7.2"` line that gets added to every `MockProject`.

```
bundler: failed to load command: tapioca (/opt/hostedtoolcache/Ruby/3.2.5/x64/lib/ruby/gems/3.2.0/bin/tapioca)
        /opt/hostedtoolcache/Ruby/3.2.5/x64/lib/ruby/gems/3.2.0/gems/bundler-2.5.11/lib/bundler/dsl.rb:130:in `gem':  (Bundler::Dsl::DSLError)
        [!] There was an error parsing `Gemfile`: You cannot specify the same gem twice with different version requirements.
        You specified: json (>= 2.7.2) and json (>= 0). Bundler cannot continue.
```